sort.go

```diff
@@ -67,10 +72,16 @@ func SortYAMLObjects(yamlBytes []byte) ([]byte, error) {
 		return objs[i].obj.GetName() < objs[j].obj.GetName()
 	})
 
-	var sorted [][]byte
-	for _, obj := range objs {
-		sorted = append(sorted, bytes.TrimSpace(split[obj.i]))
+	output := yamlsep
+	for i, obj := range objs {
+		output = append(output, bytes.TrimSpace(split[obj.i])...)
```

Comment:
If you always add the leading document separator then this code becomes simpler. E.g.

```
      output = append(output, yamlsepnl...)
      output = append(output, bytes.TrimSpace(split[obj.i])...)
```